images/envoy/main.tf

```diff
@@ -1,33 +1,31 @@
+terraform {
+  required_providers {
+    oci = { source = "chainguard-dev/oci" }
+  }
+}
+
 variable "target_repository" {
   description = "The docker repo into which the image and attestations should be published."
 }
 
-module "latest" {
-  source = "../../tflib/publisher"
-
-  name = basename(path.module)
+module "config" { source = "./configs" }
```

Comment:
super nit: I've typically been renaming these `config` because the plural seems misleading.